packages/next/src/index.ts

```diff
@@ -2690,13 +2690,14 @@ async function getServerlessPages(params: {
   outputDirectory: string;
   appPathRoutesManifest?: Record<string, string>;
 }) {
+  const appDir = path.join(params.pagesDir, '../app');
```

Comment:
```suggestion
  const appDir = path.posix.join(params.pagesDir, '../app');
```

Should this be posix? I'm pretty sure glob should be forward slash

> Note Glob patterns should always use / as a path separator, even on Windows systems, as \ is used to escape glob characters.

https://github.com/isaacs/node-glob

Reply:
This results in the following error on a Windows machine:

> Error: basePath/cwd must be an absolute path (app)